modules/seqexec/engine/src/main/scala/seqexec/engine/Engine.scala

```diff
@@ -60,14 +60,20 @@ class Engine[D, U](stateL: Engine.State[D]) {
     */
   def load(seq: Sequence[IO]): Sequence.State[IO] = Sequence.State.init(seq)
 
+  /**
+    * Redefines an existing sequence. Changes the step actions, removes steps, adds new steps.
+    */
+  def reload(seq: Sequence.State[IO], steps: List[Step[IO]]): Sequence.State[IO] =
```

Comment:
can you use `F[_]` here?

Reply:
I didn't propagate the `F[_]` to `Engine`